src/media/js/settings_app.js

```diff
@@ -19,6 +19,9 @@ define('settings_app',
         return window.location.search.indexOf('cache=false') === -1;
     }
 
+    var is_packaged_app = window.location.protocol === 'app:';
```

Comment:
i feel capability is a good place to put it given how we are using it. it describes the environment of the app.